xm-commons-domain-event/src/main/java/com/icthh/xm/commons/domain/event/domain/enums/DefaultDomainEventSource.java

```diff
@@ -0,0 +1,6 @@
+package com.icthh.xm.commons.domain.event.domain.enums;
+
+public enum DefaultDomainEventSource {
+    db,
```

Comment:
please use upper case for enum values and add `LEP` source as well.
we will use to produce events from LEPs